beetsplug/fetchart.py

```diff
@@ -470,6 +470,14 @@ class ITunesStore(RemoteArtSource):
     NAME = u"iTunes Store"
     API_URL = u'https://itunes.apple.com/search'
 
+    def __init__(self, *args, **kwargs):
+        super(ITunesStore, self).__init__(*args, **kwargs)
+        high_resolution = self._config['high_resolution'].get()
+        if high_resolution:
+            self.image_suffix = '100000x100000-999'
+        else:
+            self.image_suffix = '1200x1200bb'
```

Comment:
Because `self.image_suffix` is only used in one location (i.e., the `get` method), maybe it makes more sense to do this work here. That is, we could avoid overriding `__init__` and the `super` baggage that comes with, along with the complexity of an additional configuration field, by just computing a local `image_suffix` variable inside the `get` method. This would just mean copying and pasting the `if` condition into `get` and replacing `self.image_suffix` with just `image_suffix`.

Reply:
makes sense! Patched in 2593a5b.